tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -403,9 +403,9 @@ take effect instantly, thereby expediting your testing.
 
 positive
 : **Note:**
-The different stages of snapcraft are: "pull" (download source for all parts), "build", "stage"
+The different steps of [the snapcraft lifecycle](https://forum.snapcraft.io/t/snapcraft-lifecycle/5123) are: "pull" (download source for all parts), "build", "stage"
```

Comment:
It would be better if this link pointed to the actual docs rather than the forum